### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

File: pandas/io/sas/sasreader.py [hunk 1]
```diff
@@ -126,6 +126,10 @@ def read_sas(
     -------
     DataFrame if iterator=False and chunksize=None, else SAS7BDATReader
     or XportReader
+
+    Examples
+    --------
+    >>> df = pd.read_sas("sas_data.sas7bdat")
```
Comment (mroeschke) [inline pandas/io/sas/sasreader.py:132]:
```suggestion
    >>> df = pd.read_sas("sas_data.sas7bdat")  # doctest: +SKIP
```

File: pandas/util/_tester.py [hunk 2]
```diff
@@ -25,6 +25,11 @@ def test(extra_args: list[str] | None = None, run_doctests: bool = False) -> Non
         Whether to only run the Python and Cython doctests. If you would like to run
         both doctests/regular tests, just append "--doctest-modules"/"--doctest-cython"
         to extra_args.
+
+    Examples
+    --------
+    >>> pd.test()  # doctest: +SKIP
+    running: pytest...
```
Comment (MarcoGorelli) [inline pandas/util/_tester.py:32]: definitely a good idea to skip this one 😄

File: pandas/io/sas/sasreader.py (hunk 1 above)
Comment (DeaMariaLeon) [reply pandas/io/sas/sasreader.py]: Thank you..

**Comment by mroeschke** [comment]
`Error: /home/runner/work/pandas/pandas/pandas/core/frame.py:2739:EX01:pandas.DataFrame.to_feather:No examples section found`

**Comment by mroeschke** [comment]
Could you merge in main once more. I merged your earlier PR first

**Review by MarcoGorelli — APPROVED** [review]
thanks @DeaMariaLeon !

**Pull request merged**